nixos/modules/services/networking/pixiecore.nix

```
@@ -0,0 +1,112 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.pixiecore;
+in
+{
+  options = {
+    services.pixiecore = {
+      enable = mkEnableOption "Pixiecore";
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = true;
```

```suggestion
        default = false;
```

Except for ssh, no port should be opened by enabling a service. (If another module does, it should be fixed.)